app/views/homepage/index.html.erb

```diff
@@ -4,14 +4,17 @@
       <%= render partial: 'shared/header', locals: { isCover: true } %>
       <div class="c-hero">
         <h1 class="title -light -huge">Advancing financial inclusion</h1>
-        <p class="subtitle">A platform for <a href="#">governments</a>,
-          <a href="#">researchers</a> and <a href="#">FSP Techs.</a>
+        <p class="subtitle">A platform for
+          <a class="js-anchor" data-anchor="#governments" href="#">governments</a>,
+          <a class="js-anchor" data-anchor="#researchers" href="#">researchers</a>
+          and
+          <a class="js-anchor" data-anchor="#fsp-tech" href="#">FSP Techs.</a>
         </p>
       </div>
     </div>
   </div>
 
-  <div class="homepage-section -government" style="background-image: url(images/homepage/home-bg-1b.png)">
+  <div id="governments" class="homepage-section -government" style="background-image: url(images/homepage/home-bg-1b.png)">
```

Comment:
Can you add these ids to the components `c-card-highlight`?

It's just a matter of consistency: some of the sections have padding/margin, some don't. If we want the user to be scrolled to the exact same position of the cards, let's use something with the same properties.